src/Helpers/api.js

```diff
@@ -1,3 +1,6 @@
 import axios from "axios";
 
-export default axios.create({ baseURL: "http://localhost:8000/api/" });
+export default axios.create({
+    baseURL: "http://localhost:8000/api/",
```

Comment:
since you are planning to merge this into the live version you should put back the herokuapp url